tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -388,6 +415,17 @@ <h3>No profile data was found.</h3>
         type: Number,
         value: 1000,
       },
+      _profileNumRetry: {
+        type: Number,
```

Comment:
Here `_profileNumRetry` is declared as a number, but below you compare
it for strict equality with `""`. It seems like one of these must be
wrong—does the `paper-input` provide a string or a number?

Reply:
Yes, it is String. Also fixed for _profileDuration